packages/elements/src/pagination/__test__/pagination.test.js

```diff
@@ -305,129 +400,133 @@ describe('pagination/Pagination', () => {
     let lastButton;
 
     beforeEach(async () => {
-      el = await fixture('<ef-pagination page-size="5" total-items="32" lang="en-gb"></ef-pagination>');
+      el = await fixture('<ef-pagination max="7" lang="en-gb"></ef-pagination>');
       firstButton = el.shadowRoot.querySelector('#first');
       previousButton = el.shadowRoot.querySelector('#previous');
       nextButton = el.shadowRoot.querySelector('#next');
       lastButton = el.shadowRoot.querySelector('#last');
     });
+    afterEach(async () => {
+      el = '';
+      firstButton = '';
+      previousButton = '';
+      nextButton = '';
+      lastButton = '';
+    });
 
-    it('Should fire page-changed event when first button is click and page changes', async () => {
-      el.page = '3';
+    it('Should fire value-changed event when first button is clicked', async () => {
+      el.value = '3';
       setTimeout(() => {
         firstButton.click();
       });
-      const { detail } = await oneEvent(el, 'page-changed');
-
-      expect(el.page).to.equal('1');
+      const { detail } = await oneEvent(el, 'value-changed');
+      expect(el.value).to.equal('1');
       expect(detail.value).to.equal('1');
     });
 
-    it('Should fire page-changed event when next button is click and page changes', async () => {
-      el.page = '1';
+    it('Should fire value-changed event when next button is clicked', async () => {
       setTimeout(() => {
         nextButton.click();
       });
-      const { detail } = await oneEvent(el, 'page-changed');
-
-      expect(el.page).to.equal('2');
+      const { detail } = await oneEvent(el, 'value-changed');
+      expect(el.value).to.equal('2');
       expect(detail.value).to.equal('2');
     });
-    it('Should not fire page-changed event when next button is click but the page is already at the maximum page', async () => {
+
+    it('Should not fire value-changed event when next button is clicked but the page is already at the maximum page', async () => {
       let eventFired;
-      el.page = '7';
+      el.value = '7';
 
       setTimeout(() => nextButton.click());
 
       try {
-        await waitUntil(async () => await oneEvent(el, 'page-changed'), 'Event does not fire', { timeout: 0 });
+        await waitUntil(async () => await oneEvent(el, 'value-changed'), 'Event does not fire', { timeout: 0 });
```

Comment:
if does not have the `waitUntil`, It does not work because impossible to detect elements that do not fire any events, that is why I am using a timeout to limit the time of response. 
Do you have a better solution?

Reply:
I will go with this solution for now.